source/blog/2017-02-10-matching-fedora-ostree-released-content-with-each-2week-atomic-release.html.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Matching Fedora OSTree Released Content With Each 2 Week Atomic Release' 
+author: dustymabe
+date: 2017-02-10 15:23:32 UTC
+tags: fedora, atomic
+published: true
+comments: true
+---
+
+***TL;DR***: The default Fedora cadence for updates in the RPM streams is once a
+             day. Until now, the OSTree-based updates cadence has matched this, but
+             we're changing the default OSTree update stream to match the
+             Fedora Atomic Host image release cadence (once every two weeks).
+
+READMORE
+
+---
+
+In Fedora we release a new Atomic Host approximately every two weeks. In the
+past this has meant that we bless and ship new ISO, QCOW, and Vagrant images that
+can then be used to install and or start a new Atomic Host server. But
+what if you already have an Atomic Host server up and running? 
+
+Servers that are already running are configured to get their updates
+directly from the OSTree repo that is sitting on Fedora Infrastructure
+servers. The client will ask ***What is the newest commit for my
+branch/reference?*** and the server will kindly reply with the most recent commit.
+If the client is at an older version then it will start to pull the
+newer commit and will apply the update. 
+
+This is exactly how the client is supposed to behave, but one problem
+with the way we have been doing things in the past is that we have
+been updating everyone's "branch/reference" every night when we do
+our updates runs in Fedora.
+
+This has the side effect of meaning that users can get content as soon
+as it has been created, but it also means that the two week release
+process where we perform testing and validation really means nothing
+for these users, as they will get something before we ever did testing
+on it. 
+
+We have decided to slow down the cadence of the
+`fedora-atomic/25/x86_64/docker-host` ref within the OSTree
```

Comment:
At the risk of being pendantic, you use 'ref' for the first time here but in the question in line 26, you use 'branch/reference'.  I'd pick one and align all of the text with it.